addons/mrp/models/mrp_production.py

```diff
@@ -1894,79 +2081,3 @@ def _check_serial_mass_produce_components(self):
                 components.setdefault(move.product_id, set()).update(lot_ids)
             multiple_lot_components.update([p for p, l in components.items() if len(l) != 1])
         return (have_serial_components, have_lot_components, missing_components, multiple_lot_components)
-
-    def _generate_backorder_productions_multi(self, serial_numbers, cancel_remaining_quantities=False):
```

Comment:
In fact it would be great to keep it with a deprecation warning and a redirect to new _split_productions. Maybe it's too much but for 3 month of stable versions it's maybe better to do thing correctly (in case of someone doing custom code). If it's too annoying forget this part